packages/spectral/src/generators/componentManifest/createActions.ts

```diff
@@ -120,6 +121,7 @@ interface RenderActionProps {
     description: string;
     inputs: Input[];
     examplePayload?: unknown;
+    component: string;
```

Comment:
Could we name this `componentKey` for specificity? `component` often refers to the combination of key, version, and public.

Reply:
Key is enough here. Will name it more specifically